sdccc/src/main/java/com/draeger/medical/sdccc/tests/util/MdibHistorian.java

```diff
@@ -205,10 +224,29 @@ public HistorianResult episodicReportBasedHistory(final String sequenceId)
         try {
             final var messages =
                     messageStorage.getInboundMessagesByBodyType(Constants.RELEVANT_REPORT_BODIES.toArray(new QName[0]));
+            final BigInteger[] last = new BigInteger[1];
             final var stream = messages.getStream()
                     .map(this::unmarshallReport)
                     .filter(report -> sequenceId.equals(report.getSequenceId()))
                     .filter(mdibVersionPredicate)
+                    .filter(it -> {
+                        if (ensureUnique) {
```

Comment:
Would it be a good idea to compare the whole report and not only the mdib version? Otherwise a report with the same mdib version but different content could be filtered.

Reply:
Indeed, a check like that would make sense.
Unfortunately, the Requirements are quite open to interpretation in that respect.
My proposal would be to revisit this topic once things have been clarified.

Reply:
Update: Malte suggested to NOT filter out Reports that have the same mdibVersion and the same ReportType, but not the same Contents as the previous Report. As far as I can see, this case clearly should not happen and it would indeed be best when our Tests fail when it does, as this is a clear indication that something is wrong and the Device violated one of the Requirements Biceps:C11-C15.

Reply:
Done.